Doc/library/statistics.rst

```diff
@@ -488,13 +488,17 @@ of applications in statistics, including simulations and hypothesis testing.
 
     If *sigma* is negative, raises :exc:`StatisticsError`.
 
-    .. attribute:: mu
+    .. attribute:: mean
```

Comment:
Does this need a `versionadded` directive since they are new attributes ?

Reply:
Sorry, please ignore my comment. I thought this was changing the existing API but it was added in 3.8 and has `versionadded` in the end.